source/_base.css

```diff
@@ -43,7 +43,7 @@
   animation-duration: 3s;
 }
 
-@media (prefers-reduced-motion) {
+@media print and (prefers-reduced-motion) {
```

Comment:
This should be a logical `or`, not an `and`:

```css
@media (print), (prefers-reduced-motion) {
  ...
}
```

Reply:
Awesome! Thanks @daneden